src/docs/error-handling.dox

```diff
@@ -36,9 +36,10 @@ can get more detailed information about the last session API call. The informati
 error code, a sub-level error code and message. The intended usage of the sub-level error code
 is to allow users to programmatically make decisions on the application level.
 
-The sub-level error code and error messages are optional and default to WT_NONE and an empty
-string respectively. The error message string is owned by the session, and is invalidated after
-each session API call.
+The sub-level error code is optional and defaults to WT_NONE. When the prior API call succeeds, a
+default success message is returned. When the prior API call fails, the error message is optional
```

Comment:
I think it's worth clarifying here that if no other API call has been made, then the default error message string is the default success message.

Reply:
I have added this reference, let me know what you think.